endpoint/common.php

```diff
@@ -7,21 +7,26 @@
 require $autoload_name;
 
 use Aws\Sns\SnsClient;
+use Aws\Kinesis\KinesisClient;
+use Aws\Exception\AwsException;
 
 /*stop AWS complaining*/
 date_default_timezone_set('UTC');
 
+$GLOBALS['kinesis_stream_name'] = "encodings_endpoint_data";
```

Comment:
Would be better to set this externally, how are the other globals like `DB_HOST` set? My guess is that they come from the Cloudformation and the instance kickstart script echoes the values into `php.ini` or similar?

Reply:
Done.